app/src/main/java/com/nutomic/syncthingandroid/service/RestApi.java

```diff
@@ -157,13 +163,19 @@ public void onWebGuiAvailable() {
     }
 
     public void reloadConfig() {
-        new GetRequest(mContext, mUrl, GetRequest.URI_CONFIG, mApiKey, null, result -> {
-            Log.v(TAG, "reloadConfig: " + result);
-            mConfig = new Gson().fromJson(result, Config.class);
-            if (mConfig == null) {
-                throw new RuntimeException("config is null: " + result);
-            }
-        });
+        new GetRequest(mContext, mUrl, GetRequest.URI_CONFIG, mApiKey, null, result -> onReloadConfigComplete(result));
```

Comment:
Can't the lamba just be rewritten as this::onReloadConfigComplete

Reply:
ok